src/silverlabnwb/timings.py

```diff
@@ -0,0 +1,68 @@
+import abc
+
+import numpy as np
+import pandas as pd
+
+
+class LabViewTimings(metaclass=abc.ABCMeta):
+
+    def __init__(self):
+        self.pixel_time_offsets = None
+        self.cycle_time = None
+
+    @abc.abstractmethod
+    def _read_relative_times_file(self, file_path):
+        pass
+
+
+class LabViewTimingsPre2018(LabViewTimings):
+
+    def __init__(self, relative_times_path):
+        self._read_relative_times_file(relative_times_path)
```

Comment:
This (same for the other subclass) should call the base `LabViewTimings` constructor using `super()`, to ensure the object is properly initialised. In this case it doesn't really matter because the base class constructor only initialises some attributes which will be overwritten later!

In fact, since they both call `_read_relative_times_file` as the first step, that should maybe go into the base class `__init__` that both subclasses call - unless we think that other subclasses would not.